Lesson19/selcan/recursion.js

```diff
@@ -0,0 +1,142 @@
+// See completed in the lesson examples in the lesson files folder: https://github.com/frontend-from-0/group091123-files
+
+// 4. Write a recursive function to remove all occurrences of a specified character from a string.
+const sentenceEx5 = 'Hello, how are you?';
+const charToRemove = 'o';
+ function removeO(arr,val){
```

Comment:
Functions are supposed to be reused so it's good to give them more generic names, e.g. 'removeCharacter' as it will not always remove 'o'.

Reply:
Also, the given value is a string, not array so it's better to call the first parameter something else (e.g. 'sentence').

Reply:
I could'nt understand what you mean here;" it's better to call the first parameter something else (e.g. 'sentence')"

Reply:
I meant rename `arr` parameter that `removeO` function takes to something else, e.g. `sentence`. The parameter is expected to be in a string format, there's no reason to call it 'arr'.

Make sure to update the code so that the new name is used.